src/components/News.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+    <b-container fluid >
+    <h5>News</h5>
+    <b-col lg="8" offset-lg="2">
+        <b-card v-for="item in news" :key="item.article" class="m-lg-3">
+            <b-icon icon="link45deg"></b-icon><a :href="item.link">{{ item.article }}</a>
+        </b-card>
+    </b-col>
+    </b-container>
+</template>
+
+<script>
+export default {
+  name: 'News',
+  data() {
+    return {
+      news: [
```

Comment:
is there a goal to store this in the db at some point?

Reply:
probably something to discuss - I don't think news items will be very interesting to the audience after the bar reopens - right now the bar is closed and will only reopen once development is done on the new location so this components relevance might be short lived and therefore not worth building into the db